ProfessorAssessor/Models/Course.m

```diff
@@ -1,12 +1,28 @@
 #import "Course.h"
+#import "Networker.h"
 
 @implementation Course
 
+@dynamic identifier;
+@dynamic createdAt;
+@dynamic updatedAt;
 @dynamic name;
 @dynamic reviews;
 
 + (nonnull NSString *)parseClassName {
     return @"Course";
 }
 
++ (Course *)courseFromPFObject:(PFObject *)object {
+    if (object) {
+        return [Networker courseFromObject:object
```

Comment:
I don't understand. Course shouldn't know about networker. Food for thought. What if you wanted to extract course into a library that's used by multiple apps. Each will have their own networker and way of getting the information needed to create a course.